packages/webview_flutter/webview_flutter_android/android/src/main/java/io/flutter/plugins/webviewflutter/WebViewFlutterPlugin.java

```diff
@@ -33,12 +36,43 @@
  * <p>Call {@link #registerWith} to use the stable {@code io.flutter.plugin.common} package instead.
  */
 public class WebViewFlutterPlugin implements FlutterPlugin, ActivityAware {
-  private InstanceManager instanceManager;
+  @Nullable private InstanceManager instanceManager;
 
   private FlutterPluginBinding pluginBinding;
   private WebViewHostApiImpl webViewHostApi;
   private JavaScriptChannelHostApiImpl javaScriptChannelHostApi;
 
+  /**
+   * Retrieves the {@link WebView} that is associated with `identifer`.
+   *
+   * <p>See the Dart method `AndroidWebViewController.webViewIdentifier` to get the identifier of an
+   * underlying `WebView`.
+   *
+   * @param engine the execution environment the {@link WebViewFlutterPlugin} should belong to. If
+   *     the engine doesn't contain an attached instance of {@link WebViewFlutterPlugin}, this
+   *     method returns null.
+   * @param identifier the associated identifier of the `WebView`.
+   * @return the `WebView` associated with `identifier` or null if a `WebView` instance associated
+   *     with `identifier` could not be found.
+   */
+  @SuppressWarnings("unused")
+  @Nullable
+  public static WebView getWebView(FlutterEngine engine, long identifier) {
```

Comment:
Putting the accessor on this class encourages people to view this whole class as public, stable API, which could easily lead to accidental breakage later. Can we make a new wrapper class to be the public API, with just this method?